



public class Main{
public 
    /*
    Для того чтобы возбуждать исключения необходимо в шапке функции написать о том какие исключения мы будем возбуждать 
    Конструкция:
    "что возращает функция" "название функции"("аргументы функции") throws "что подымает функция"{
        "Тело функции"
    }
    */
    static void main(String args[]) throws StackOverflowError, OutOfMemoryError{
        /*
        Конструкция throw
        throw new "название исключения"("коментарий к исключению (если поддерживается)")
        Данная конструкция необходима чтобы подымать исключения
        */
        throw new StackOverflowError("Коминтарий для исключения"); 
    }
}
